app/channels/application_cable/connection.rb

```diff
@@ -8,11 +8,9 @@ def connect
 
     protected
     def find_verified_user
-        if verified_user = User.find_by(id: cookies.signed['user.id'])
-          verified_user
-        else
-          reject_unauthorized_connection
-        end
+      verified_user = User.find_by(id: env[‘warden’].user.id)
+      return reject_unauthorized_connection unless verified_user
+      verified_user
```

Comment:
こちらで問題なく `current_user` が取れていますよ。

```suggestion
      session_key = cookies.encrypted[Rails.application.config.session_options[:key]]
      verified_id = session_key['warden.user.user.key'][0][0]
      User.find_by(id: verified_id)
```